### ISSUE TITLE
Better bot messages

### ISSUE THREAD

**Issue by Fubinator** [Fubinator/planning-poker-discord-bot#4]
The bot messages should be updated as they are sometimes difficult to understand.

**Comment by rgroves** [comment]
I can pick this one up if it's still unassigned.

Could anyone help on getting this to run in Discord to test changes?

I have already created an app on the Discord Developer Portal and have a Bot Token for the .env file. Then what? Sorry, have never developed or worked on a Discord bot before this. Thanks!

Edit: Disregard about the help I believe I have it sorted and mentioned steps I took in #27 in hopes that can be added for future contributors who may have never worked with a Discord bot.
